### DISCUSSION TITLE
Audio Quality

### DISCUSSION THREAD

**Discussion by ApoAlKasem1** [deniscerri/ytdlnis#8] [General]
I don't understand how to increase the audio quality

**Comment by deniscerri** [comment]
![Screenshot_20221224-092603_YTDLnis](https://user-images.githubusercontent.com/64997243/209427760-99b75c18-4b01-4023-b9ff-3717ea9eebb3.png)
Put it on 0 for best quality

**Reply by prateekpatel1432** [reply]
From right side of screen

**Comment by GarstigeLady** [comment]
Und wie kann ich die kbits erhöhen außer 128kbit
